Main-rpn/Main-rpn/Main-rpn.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <fstream>
+
+#include <math.h>
+#include <string>
+#include <stack>
+
+int calculate(const std::string &infix)
```

Comment:
I think this function is too long and it has to do 2 jobs, infix to postfix conversion and postfix calculation. So can we split this function into 2 functions?